massa-versioning-worker/src/versioning.rs

```diff
@@ -461,18 +461,27 @@ impl<const N: usize> TryFrom<[(MipInfo, MipState); N]> for MipStore {
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub enum UpdateWithResult {
+    Success(Vec<MipInfo>, Vec<MipInfo>),
```

Comment:
Add a comment to understand the "update"/"added" order here (otherwise we don't understand what the two Vec<MipInfo> are).